docs/share_channels.rst

```diff
@@ -74,4 +74,4 @@ Make Content Channels Available for Import into Kolibri
 Import Content Channels into Kolibri
 ************************************
 
-Follow the instructions to `import content channels into Kolibri <http://kolibri.readthedocs.io/en/develop/user/manage.html#import-content-channel-to-kolibri>`_.
\ No newline at end of file
+Follow the instructions how to `import content channels into local installations of Kolibri <http://kolibri.readthedocs.io/en/release-v0.7.x/manage.html#import-content-into-kolibri>`_.
```

Comment:
We should try to start linking to `en/latest`, otherwise we'll need to keep updating this.

(I see that `latest` is not actually the latest, so you can ignore this comment for now.)

Reply:
Maybe the `latest` should be the `latest major public` release, and leave `develop` for... latest intermediate changes? 

`develop` still shows last-year version of the docs, is the merge imminent?

Reply:
> Maybe the latest should be the latest major public release

Agreed, for user docs. We are supposed to merge that to `master` so latest should track `master`... except I haven't merged to master recently

For developer docs it's a little different and `latest` can track `develop.

> develop still shows last-year version of the docs, is the merge imminent?

I have a lot of branch cleanup to do. Let's call it imminent eventually. Maybe later next week

Reply:
Agreed ref. difference user docs vs. developer docs. 
I applied the stop-gap solution in the RTD dashboard, and reverted the link towards the `latest`.

NP for the merge, I know it's a delicate operation, so better safe than sorry 😉